grails-app/controllers/au/org/ala/profile/hub/ProfileController.groovy

```diff
@@ -409,9 +409,9 @@ class ProfileController extends BaseController {
                 ImageType type = params.type as ImageType
                 //NB this imageId param already has the file extension on it, really the file name on disk
                 if (type == ImageType.STAGED) {
-                    displayLocalImage("${grailsApplication.config.image.staging.dir}/", params.opusId, params.profileId, params.imageId, true)
+                    displayLocalImage("${grailsApplication.config.image.staging.dir}/", params.opusId, params.profileId, params.fileName, true)
```

Comment:
do not rename this parameter. keep original. It is used by other parts of application.